File: mini-commerce/src/main/java/me/hero/minicommerce/item/adapter/in/web/dto/ModifyItemResponse.java
package me.hero.minicommerce.item.adapter.in.web.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import me.hero.minicommerce.item.application.port.in.dto.ModifiedItemDto;

@Data @NoArgsConstructor
public class ModifyItemResponse {
  private Long id;
  private String name;
  private Long price;
  public ModifyItemResponse(ModifiedItemDto dto) {
    this.id = dto.getId();
    this.name = dto.getName();
    this.price = dto.getPrice();
  }
}

File: mini-commerce/src/main/kotlin/me/hero/minicommerce/item/adapter/in/web/dto/ModifyItemResponse.kt
package me.hero.minicommerce.item.adapter.`in`.web.dto

import me.hero.minicommerce.item.application.port.`in`.dto.ModifiedItemDto

data class ModifyItemResponse(
    val id: Long,
    val name: String,
    val price: Long,
) {
    constructor(dto: ModifiedItemDto) : this(
        id = dto.id,
        name = dto.name,
        price = dto.price,
    )
}